test/test_jit.py

```diff
@@ -1446,6 +1446,41 @@ def test_fuser_multiple_blocks(this, that, theother, meme):
 
         self.assertEqual(cu.test_fuser_multiple_blocks(*inputs), outputs)
 
+    def test_python_new_trace(self):
+
+        eg = torch.zeros(1, 2, 3, requires_grad=True)
+
+        @torch.jit.trace(eg)
+        def foo(x):
+            x = torch.neg(x)
+            return F.dropout(x)
```

Comment:
Can you please use a custom Python function instead of Dropout? It's going to break those tests once someone moves it to C++ and it's not their fault.

Reply:
There are two tests here. One tests if dropout export works, which was the original reason for this PR. The other is a more generic test that tracing a Python function from a new trace works. I updated the names but both are useful tests.